tools/bootstrap-librdkafka.sh

```diff
@@ -34,7 +34,12 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 
 ./configure --clean
 make clean
-./configure --install-deps --source-deps-only --prefix="$INSTALLDIR"
+
+if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
nit: the quotes are not needed with double-bracket statements.

Reply:
ack